src/assets/properties-panel.css

```diff
@@ -1237,7 +1237,7 @@ textarea.bio-properties-panel-input {
   --feel-popup-close-background-color: hsla(219, 99%, 53%, 1);
   --feel-popup-gutters-background-color: hsla(0, 0%, 90%, 1);
 
-  position: absolute;
+  position: fixed;
```

Comment:
For context: I added this to better work on top of other components (cf. https://github.com/camunda/camunda-modeler/issues/3877#issuecomment-1733198745)

Reply:
:+1: